requirements.txt

```diff
@@ -1,2 +1,2 @@
-ansible-core==2.13.7
-netaddr==0.8.0
+ansible-core==2.18.1
```

Comment:
this might be requiring python 3

Reply:
Do you mean on the remote machine? Im seeing this inside the KIB container:
```
/opt # ansible --version | grep "python version"
  python version = 3.10.15 (main, Sep 11 2024, 09:20:18) [GCC 12.2.1 20220924]
/opt # python --version 
Python 3.10.15
```